code/game/objects/items/reagent_containers/food/snacks.dm

```diff
@@ -2074,6 +2074,19 @@
 	bitesize = 2
 	tastes = list("bread" = 10, "cheese" = 10)
 
+/obj/item/reagent_containers/food/snacks/marinebread //meme bread for breadify smite
+	name = "Bread"
+	desc = "Some plain old Earthen bread. An air of penance surrounds it."
+	icon_state = "breadtg"
+	list_reagents = list(/datum/reagent/consumable/nutriment = 60)
+	filling_color = "#FFF896"
+	bitesize = 2
+	tastes = list("guilt" = 1, "salt" = 1)
+
+/obj/item/reagent_containers/food/snacks/marinebread/Destroy() //delete the marine trapped inside, tasty!
+	for(var/i in contents)
+		qdel(i)
+	return ..()
```

Comment:
this is already done on /atom/movable/Destroy

Reply:
By default reagent_containers/food/snacks/Destroy() will place any items inside on the floor. The breadify smite qdels the human target on regular TG so I overrode /food/snacks/Destroy() to get that behavior.
If there's a better way of doing it I'm all ears.

Reply:
I guess I could delete the Destroy() that dropped contents to turf, but I was trying to preserve the original behavior.